src/components/cards/WrittenReviewCard.tsx

```diff
@@ -23,10 +23,8 @@ export default function WrittenReviewCard({ myReview }: ReviewCardProps) {
         className="lg:w-[68.6rem] lg:h-[34.6rem] md:w-[60rem] md:h-[20.8rem] sm:w-[32.7rem] sm:h-[20.8rem] rounded-[2.4rem] lg:px-[2.4rem] lg:py-[3.2rem] md:px-[2rem] md:py-[2rem] sm:px-[1.4rem] sm:py-[2rem] bg-white border-none flex flex-col shadow-custom3"
       >
         <div className="flex flex-row items-center lg:gap-[1.2rem] md:gap-[0.8rem] sm:gap-[0.8rem] lg:w-[64rem] ">
-          <MovingTypeChips
-            type={myReview.estimation.moveInfos.serviceType}
-            specificRequest={myReview.estimation.moveInfos.isSpecificRequest}
-          />
+          <MovingTypeChips type={myReview.estimation.moveInfos.serviceType} />
+          {myReview.estimation.moveInfos.isSpecificRequest === true && <MovingTypeChips type="APPOINTMENT" />}
```

Comment:
```suggestion
          {myReview.estimation.moveInfos.isSpecificRequest && <MovingTypeChips type="APPOINTMENT" />}
```
이렇게만 수정하시면 될 것 같습니다! 수고 많으셨습니다

Reply:
넵 수정했습니다!